tests/test_cli_download_files.py

```diff
@@ -54,6 +54,7 @@ def test_dry_run_from_doi_wrong():
     assert test_result.exit_code == 2
 
 
+@pytest.mark.skip(reason="tests: skipping temporary failure")
```

Comment:
This PR skips 9 tests, which is perhaps too much... Wasn't only 1 test usually failing? It might be better to skip only that one single offending one... 

- this PR: === 45 passed, 9 skipped in 12.64s ===
- master: === 54 passed in 14.22s ===

Currently on my machine `master` works fine :)

Reply:
All the tests were failing where we are checking the file size. On my machine everything runs fine as well.

Reply:
So perhaps rebase on of your "real PRs" and let's merge that one first :) and keep this one for later improvements.

Reply:
I had a look at the reproducing the failures locally so that we don't have to skip so much tests.

A recipe to get the failure locally is:

```console
$ for i ({1..100}) cernopendata-client download-files --recid 3005 --verify || exit
```

When the failure happens, the ouput file is actually a static web page that the server sends back in case of problems:

```console
$ head -5 3005/0d0714743f0204ed3c0144941e6ce248.configFile.py
<!DOCTYPE html>
<html lang="en">
<head>
  <meta http-equiv="content-type" content="text/html; charset=utf-8" />
  <meta http-equiv="Content-Language" content="en" />
```

So what we could do is for each test, when the failure happens, to check whether we have this situation, and if so, pass over the problem by skipping it, or try to re-download the same item say 10 times, which usually works. This fix could be done "inside" the downloader component.

Here is how to recognise the situation:

- The downloaded file is exactly 3846 bytes long.  IIRC this is exactly what we have been seeing in the past.

If we would like to be more precise, in addition to testing the output size 3846 bytes, we could do two more tests:

- The downloaded file contains a text string "THIS IS THE DEFAULT FRAMEWORK CSS" or "European Organization for Nuclear Research" which are both part of the error page text.  (This text is probably here to stay.)

- The downloaded file checksum is `adler32:a82d5324`. (If we would like to be ultra strict; we would need to update this checksum in case the error page design changes.)

P.S. Dunno whether server returns some error code (such as 403) and whether our downloader pays attention to it?

P.S. Obviously, we'd also address these problems on the server side...